aries-site/src/layouts/main/Footer.js

```diff
@@ -40,28 +35,26 @@ export const Footer = () => {
         <Box>
           <Text>&copy; {year} Hewlett Packard Enterprise</Text>
         </Box>
-        <Box direction='row'>
-        {externalFooterLinks.map(item => (
-          <Link
-            key={item.name}
-            href={item.href}
-            passHref
-          >
-            <NavButton
-              item={item.name}
-              active={router.pathname === (item.href)}
-              target="_blank"
-              rel="noreferrer noopener"
-            />
-          </Link>
-        ))}
-        {/* Need to pass href because of:
+        <Box direction="row" gap="xsmall">
+          {externalFooterLinks.map(item => (
+            <Box align="start">
+              <Link key={item.name} href={item.href} passHref>
```

Comment:
@taysea I think we need to move this `key` up to the parent Box. Now getting a console warning:
![Screen Shot 2020-04-08 at 3 02 41 PM](https://user-images.githubusercontent.com/1756948/78833426-17dd4600-79aa-11ea-9aa1-661a62cd32d2.png)